ci-operator/step-registry/openshift-qe/rosa-loaded-upgrade-and-postcheck/openshift-qe-rosa-loaded-upgrade-and-postcheck-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: openshift-qe-rosa-loaded-upgrade-and-postcheck
+  steps: 
+  - ref: openshift-qe-rosa-loaded-upgrade-and-postcheck
```

Comment:
I think this ref and the related ref file name and command.sh file do not need to contain '-and-postcheck', correct me if there is any rule that must make the files under the folder consistent.

Reply:
This changed by Sharada's comments， but it seems a little confusion for the title @svetsa-rh， do we need to keep "--and-postcheck"， we can just name it to rosa-loaded-upgrade， the postcheck will be done by default， the reason I separate rosa-upgrade and post-check is the two script need to run in different container image。

Reply:
BTW, the chain, command.sh and ref file name must consistent with folder name

Reply:
I have rollback to rosa-loaded-upgrade now, thanks!